pages/1.13/developing-services/index.md

```diff
@@ -13,13 +13,13 @@ The Mesosphere Distributed Cloud Operating System (DC/OS) provides the optimal u
 
 # <a name="universe"></a>Package Repositories
```

Comment:
Should this be changed to `Catalog`

Reply:
It's an internal reference. I think we can keep it, because there may be links outside of this page that link to this section. Changing the anchor name would break any such links.